public/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from "react";
+import {
+  EuiButton,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiOverlayMask,
+  EuiButtonEmpty,
+  // @ts-ignore
+} from "@elastic/eui";
+
+interface ConfirmationModalProps {
+  title: string;
+  bodyMessage: string;
+  actionMessage: string;
+  onClose: () => void;
+  onAction: () => void;
+}
+
+const ConfirmationModal: React.SFC<ConfirmationModalProps> = ({ title, bodyMessage, actionMessage, onClose, onAction }) => {
```

Comment:
Any reason why not using `EuiConfirmModal` https://elastic.github.io/eui/#/layout/modal?

Reply:
Ah didn't see that, let me check if I can integrate that one instead. Thanks!

Reply:
Did not play well with the ModalContext that is being used to render the custom modals. Created GitHub issue:
https://github.com/opendistro-for-elasticsearch/index-management-kibana-plugin/issues/37

Will look into swapping these out after.